beets/ui/__init__.py

```diff
@@ -648,6 +780,269 @@ def term_width():
     return width
 
 
+def split_into_lines(string, width_tuple):
```

Comment:
Not a mandatory thing, but this looks like a function which it would be nice to have a unittest for, which checks a few of the edge case that the previous version of the code (as mentioned in the code comment below) failed to handle properly.

Reply:
This turned out to be important! By implementing a unit test I found errors in the colored text handling, which I have now fixed.